content/_code-samples/submit-and-verify/py/submit-and-verify.py

```diff
@@ -0,0 +1,38 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet
+from xrpl.transaction import safe_sign_and_submit_transaction
+from xrpl.core import keypairs
+from xrpl.wallet import Wallet
+
+myAddr = "rfaNymVS1fEREj4FpNroXVEcVVMph6k2Mt"
+mySeed = "ss3GZzmyEBxvwB3LUv-----------"
+
+# Derive and initialize wallet
+public, private = keypairs.derive_keypair(mySeed)
+wallet_from_seed = Wallet(mySeed, 0)
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Construct AccountSet transaction
+tx = AccountSet(
+    account=myAddr,
+    fee="10",
+    set_flag=1  # Numerical Value: 1 = asfRequireDest
+)
+
+# Sign the transaction locally & submit transaction and verify its validity on the ledger
+my_tx_payment_signed = safe_sign_and_submit_transaction(transaction=tx, wallet=wallet_from_seed, client=client)
```

Comment:
```suggestion
my_tx_payment_signed = safe_sign_transaction(transaction=tx, wallet=wallet_from_seed)
response = send_reliable_submission(transaction=my_tx_payment_signed, client=client)
result = response.result["engine_result"]
```

Reply:
The purpose of this bounty is to wait for the outcome of the transaction. `safe_sign_and_submit_transaction` just submits the txn to the network, but does not wait if the transaction got validated. Use `send_reliable_submission` for that.

Reply:
Went through the xrpl-py repo and ye, seems like it. Good catch 👍🏼 !